<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgotDialog</class>
 <widget class="QDialog" name="ForgotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Forgot to mark sinking on navigation map?</string>
  </property>
  <widget class="QLabel" name="lblStatus">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>230</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>326</width>
     <height>266</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="lblHeading">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Forgot to mark a sunken ship with the pencil on the navigation map in game? Fill this out...&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Sunken Ship Details&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lblCareer">
          <property name="text">
           <string>Career:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboCareer"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lblPatrol">
          <property name="text">
           <string>Patrol:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboPatrol"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="lblClass">
          <property name="text">
           <string>Sunken Ship Class:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboClass"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="lblDate">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Estimated Date Sunk (yyyyMMdd):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtDate"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="lblLong">
          <property name="text">
           <string>Estimated Sinking Longitude (meters):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtLong"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="lblLat">
          <property name="text">
           <string>Estimated Sinking Latitude (meters):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtLat"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="btnExit">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>128</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnRun">
        <property name="text">
         <string>Run DC Script</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>lblHeading</zorder>
  <zorder>lblStatus</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnExit</sender>
   <signal>clicked()</signal>
   <receiver>ForgotDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
